### ISSUE TITLE
Latest version is not rendering html on the build page

### ISSUE THREAD

**Issue by reubenfirmin** [jenkinsci/git-changelog-plugin#29]
When reporting a bug, please try to provide as much information as possible.

 * Plugin version used. 2.10
 * Jenkins version used. 2.121.3

I just updated to 2.10. Previously rendering templates (a very lightly edited version of your default) is now being escaped instead of rendering, i.e.. is rendering as:

```
<h1> Git changelog </h1> <p> This is different from develop thusly: </p> <h2> Unreleased </h2> <h2> Jira EV-8400 </h2> <a href="https://repo.ourrepo.io/dev/javaapis/commit/af104092266bdf6">af104092266bdf6</a> Dev Name<i>2018-09-25 01:47:44</i> <p> <h3>EV-8400 fix broken tests</h3> </p>
```

**Comment by tomasbjerre** [comment]
This was changed in 2.7. So downgrade to 2.6 or wait for an update to fix this.

**Comment by tomasbjerre** [comment]
Releasing a fix in 2.11 now. Should be available in update center within a couple of hours.
